pkg/config/testdata/application/ecs-app.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: pipecd.dev/v1beta1
+kind: ECSApp
+spec:
+  input:
+    serviceDefinitionFile: path/to/servicedef.yaml
+    taskDefinitionFile: path/to/taskdef.yaml
```

Comment:
very very nits:
```suggestion
    serviceDefinitionFile: /path/to/servicedef.yaml
    taskDefinitionFile: /path/to/taskdef.yaml
```

Reply:
To emphasize that it must be an absolute path, just in case.

Reply:
I accepted your suggestion!